magento/magento2
# fixed Notification page Select Visible items issue

Fixed issue - magento#19285
On Notification page Select All and Select Visible both works same




### Description (*)
Fixed issue [#19285](https://github.com/magento/magento2/issues/19285) On Notification page Select All and Select Visible both works same

### Fixed Issues (if relevant)
[#19285](https://github.com/magento/magento2/issues/19285) On Notification page Select All and Select Visible both works same

### Manual testing scenarios (*)
1. Login to Magento Admin.
2. Select Notifications (Bell Icon from right top of the screen).
3. Click on "See All".
4. Default paging is set to 20 rows per page.
5. Make sure you have more than 20 notifications so that you can have paging available and some notification are on next pages.
6. Now from Mass Action dropdown select "Select All" action. This will select only visible notifications (currently 20, as 20 rows per page is setup) instead of all available records.

### Contribution checklist (*)
